### PR TITLE
Token list sorting

### PR DESCRIPTION
**Pull request by kajes**
Implements list sorting for the `tokens list` command. Similar to the appliance list sorting, it uses the `--order-by` and `--descending` flags, where the former accepts a comma separated list of sortable keywords, while the latter reverses the sort order if set.

Also included are some minor build issues found:
* add missing docs for the `maintenance status`
* set correct permissions for included files in deb/rpm package

### CODE REVIEW

File: pkg/docs/maintenance.go [hunk 1]
```diff
@@ -15,6 +15,11 @@ var (
 		Long:  ``,
 	}
 
+	MaintenanceStatusDoc = CommandDoc{
+		Short: "View maintenance status of appliances",
+		Long:  "",
```
Comment (kosmas-valianos) [inline pkg/docs/maintenance.go:20]: That's a `string` I guess then default value is already `""`

**Review by kosmas-valianos — COMMENTED** [review]
LGTM but for logic checks better @danijeel to check as well

File: cmd/token/token.go [hunk 2]
```diff
@@ -26,9 +28,10 @@ func NewTokenCmd(f *factory.Factory) *cobra.Command {
 	}
 
 	var tokenCmd = &cobra.Command{
-		Use:   "token",
-		Short: "Perform actions on Admin, Claims and Entitlement tokens",
-		Long:  `The token command allows you to renew or revoke tokens used in the Collective.`,
+		Use:     "token",
+		Aliases: []string{"tokens"},
```
Comment (danijeel) [inline cmd/token/token.go:32]: Let's keep it consistent with all the other commands, and not put plural on our noun commands.

**Comment by danijeel** [comment]
since this PR includes Debian fixes, I will assign @userappgate to this PR

**Comment by userappgate** [comment]
You expect me to look at a completely non standard and having nothing to do with debian `.goreleaser.yml` file and understand if it's correct or not? :D

**Review by userappgate — APPROVED** [review]
It seems to install fine and not have man pages listed as config files.

**Pull request merged**